onnxruntime/__init__.py

```diff
@@ -9,8 +9,9 @@
 __version__ = "1.2.0"
 __author__ = "Microsoft"
 
-from onnxruntime.capi._pybind_state import get_all_providers, get_available_providers, get_device, RunOptions, SessionOptions, set_default_logger_severity, NodeArg, ModelMetadata, GraphOptimizationLevel, ExecutionMode, OrtDevice, SessionIOBinding
+from onnxruntime.capi._pybind_state import get_all_providers, get_available_providers, get_device, RunOptions, SessionOptions, set_default_logger_severity, NodeArg, ModelMetadata, GraphOptimizationLevel, ExecutionMode, OrtDevice, SessionIOBinding, TrainingParameters
 from onnxruntime.capi.session import InferenceSession, IOBinding
+from onnxruntime.capi.training_session import TrainingSession
```

Comment:
it's possible that TrainingParameters and TrainingSession won't be available in a non-training build, right? the way i worked around it was to catch the ImportError below. i wonder if there's a better way to do that - is it possible to have a separate `training/__init__.py` with the training imports?

Reply:
Also, this is duplicated with the try below